templates/admin-options.php

```diff
@@ -0,0 +1,1189 @@
+<?php
```

Comment:
This file looks to be a duplicate of the main class of the plugin. Is it necessary? Are there real admin options we are exposing?